pkg/plugins/if_test.go

```diff
@@ -145,4 +146,83 @@ var _ = Describe("Conditionals", Label("conditionals"), func() {
 			Expect(err).ShouldNot(HaveOccurred())
 		})
 	})
+	Describe("IfFiles", func() {
+		It("Fails with unknown check type", func() {
+			stage := schema.Stage{
+				IfFiles: map[schema.IfCheckType][]string{
+					schema.IfCheckType("unknown"): {},
+				},
+			}
+			err = IfFiles(logrus.New(), stage, fs, &testConsole)
+			Expect(err).Should(HaveOccurred())
```

Comment:
Better use Should(MatchError()) (https://onsi.github.io/gomega/#matcherrorexpected-any) to make sure we got the error we expected.